docs/components/modeler/bpmn/bpmn.md

```diff
@@ -0,0 +1,46 @@
+---
+id: camunda-modeler-bpmn
+title: BPMN in Camunda Modeler
+---
+
+# Quickstart
+
+## Create new BPMN 2.0 Diagram
+
+![New BPMN Diagram](./img/quickstart-1.png)
+
+To start modeling, create a new BPMN 2.0 diagram for the Camunda Platform by selecting **Create diagram > Create new BPMN diagram (Camunda Platform)** in the top-level menu.
+
+:::note
+BPMN diagrams must be created for the process engine they intend to be deployed on. You cannot run a BPMN diagram modeled for Camunda Platform in Camunda Cloud, or vice versa, at this time.
+:::
+
+## Start Modeling
+
+![Start Modeling](./img/quickstart-2.png)
+
+Now you can start to create a BPMN 2.0 model. Add the desired elements from the palette on the left hand side by dragging and dropping them onto the diagram canvas. Alternatively, you can add new elements by using the context menu that appears when you select an element in the diagram. Using the wrench icon in the context menu, you can change the type of an element in place.
+
+## Demo
+
+![Demo](./img/demo.gif)
+
+The demo above shows how to create more BPMN 2.0 elements like lanes, task types and event definitions.
+
+## BPMN 2.0 Coverage
+
+The Camunda Modeler covers all BPMN 2.0 elements for modeling processes and collaborations.
```

Comment:
Can you add a link to the BPMN Coverage for Cloud? 

/docs/reference/bpmn-processes/bpmn-coverage/